packages/framework/src/parser.ts

```diff
@@ -109,6 +120,11 @@ export const EnsembleParser = {
       customWidgets,
       home: menu ?? screens[0],
       theme,
+      themes: isArray(themes)
+        ? (Object.assign({}, ...themes) as {
+            [key: string]: EnsembleThemeModel;
+          })
+        : themes,
```

Comment:
Why not just create the object in the first place instead of converting to array then object?